COLORTHEME-3: Cotton-Candy makes panel-headings unreadable

fiannestoel opened the pull request:
Issue [COLORTHEME-3](https://jira.xwiki.org/browse/COLORTHEME-3)

old:
![unreadable_navigation_bar](https://user-images.githubusercontent.com/23619087/114193677-0dec7b00-994f-11eb-8d9b-c2651ceef2c0.PNG)
new:
![new_navigation_bar](https://user-images.githubusercontent.com/23619087/114193721-18a71000-994f-11eb-8e18-198017bf4a01.PNG)

The text was unreadable because it was white text on a white background. Looking at the Iceberg color theme as an example, I have changed the background color of the collapsed panel header (e.g. "Look & Feel") to be blue. It then didn't look right to have the active list-group-item (e.g. "Themes") also be blue, so I changed that to be gray in the same way that that is done in the Iceberg color theme.

I have visually checked for side effects as I do not 100% understand how the variables are used, and I only saw other changes in the drawer menu on the right:

old:
![old_administrator_bar](https://user-images.githubusercontent.com/23619087/114196673-e1862e00-9951-11eb-9b99-c1de668f46aa.PNG)
new:
![new_administrator_bar](https://user-images.githubusercontent.com/23619087/114196696-e6e37880-9951-11eb-9574-e0797061294d.PNG)

which also makes the "Log-out" text readable.

I hope these changes are okay, since they change the looks a bit by changing the background color.

fiannestoel committed:
change panel heading color to default

mflorea commented:
Would be good to post some before and after screenshots. Also, make sure there are no side effects (e.g. if that variable is used elsewhere).

fiannestoel committed:
change text back to white and adjust other colors

fiannestoel commented:
> Would be good to post some before and after screenshots. Also, make sure there are no side effects (e.g. if that variable is used elsewhere).

I have updated the pull request! I have changed the code too: it felt more logical to keep the text white and change the background color, as this is what it looks like in a few other color themes, like Iceberg.

mflorea commented:
The grey user name on the blue background is hard to read in the drawer.

fiannestoel committed:
change user name color in drawer

fiannestoel commented:
> The grey user name on the blue background is hard to read in the drawer.

I have changed it to white!
old:
![new_administrator_bar](https://user-images.githubusercontent.com/23619087/114756387-fa7c4e00-9d5a-11eb-9c47-9a751d32feb7.PNG)
new:
![new_drawer](https://user-images.githubusercontent.com/23619087/114756404-ff410200-9d5a-11eb-8a70-e2d7a45bb857.PNG)

Setting the navbar-default-color variable to white solved it. It does not change anything else that I am aware of, as the text in the navigation bar is also already white anyhow.

mflorea commented:
Thanks for your contribution.
